code-studio/src/js/clientState.js

```diff
@@ -89,17 +89,20 @@ clientState.levelProgress = function (scriptName, levelId) {
  * Returns the "best" of the two results, as defined in apps/src/constants.js.
  * Note that there are negative results that count as an attempt, so we can't
  * just take the maximum.
+ *
+ * Results larger than 100 are reserved for server-dependent changes and can't
+ * be cached locally.
  * @param {Number} a
  * @param {Number} b
  * @return {Number} The better result.
  */
 clientState.mergeActivityResult = function (a, b) {
   a = a || 0;
   b = b || 0;
-  if (a === 0) {
+  if (a === 0 || a > 100) {
```

Comment:
Is it 100 because of the constant here -https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/constants.js ? Can we reference that instead of the magic number?